docs/content/providers/docker.md

```diff
@@ -89,6 +89,7 @@ and [Docker Swarm Mode](https://docs.docker.com/engine/swarm/).
         deploy:
           labels:
             - traefik.http.routers.my-container.rule=Host(`example.com`)
+            - traefik.http.routers.my-container.service=my-container-service
```

Comment:
```suggestion
```

Reply:
@ldez @bluepuma77 @mmatur With ceed9c579, I tried to improve the wording and keep current "magic" example. I also added a complete example in service definition section.

Reply:
Implemented with https://github.com/traefik/traefik/commit/ceed9c579696f26fe6fc54059619775671e27a69